Pixel Game/Assets/Scripts/PlayerScripts/PlayerInventory.cs

```diff
@@ -43,119 +33,48 @@ private void Awake()
             dataToPassGameObject = GameObject.FindGameObjectWithTag("PassData");  
         }
 
-        inventoryScreenGameObject = prefabCanvas.transform.GetChild(1).gameObject; // Get the second index gameobhject which is the inventoryscreen
-        inventorySlotsTransform = inventoryScreenGameObject.transform.GetChild(0).transform; // Get the transform of inv screens child index 6 which is the InventorySlots gameobjects transform
+        var inventoryScreenTransform = GameObject.Find("InventoryScreen").transform;
+        var inventorySlotsTransform = inventoryScreenTransform.GetChild(0).transform;
         slots = inventorySlotsTransform.GetComponentsInChildren<InventorySlot>();
+        equipmentSlots = inventoryScreenTransform.GetChild(1).transform.GetComponentsInChildren<InventorySlot>();
 
-        equipmentSlots = inventoryScreenGameObject.transform.GetChild(1).
-            transform.GetComponentsInChildren<InventorySlot>(); // Get all equipment slots in player inventory
-
-        screenTabs = prefabCanvas.transform.Find("GuiTabsButtons").gameObject;
-        shopScreen = GameObject.Find("ShopScreen");
-        // Player is set in awake since it needs to be called before the start in shopscreen 
-        // since shopscreen uses shopscreen.close() 
-        playerGameObject = GameObject.FindGameObjectWithTag("MyPlayer");
         dataToPass = dataToPassGameObject.GetComponent<DataToPassBetweenScenes>();
-
-
     }
+
     void Start()
     {
         mainCamera = GameObject.FindWithTag("MainCamera");
         LoadInvGameObjectOnStartScene();
         LoadEquipmentOnStartScene();
-        ClosingUI();

```

Comment:
Does the hover toolbar get stopped on start without this? :)

Reply:
Nope...🧐
Every screen has its own Close method that gets called in its own Start, but it does not close the hover window, so I should fix that.

Reply:
Great!

Reply:
Already fixed. Will push before we merge but trying to fix another bug first related to equipment on scene transitions.

Reply:
Ok done, hapefully everything should work now :)